README.md

```diff
@@ -19,19 +19,69 @@ npm install ssb-tribes2
 - Requires **Node.js 12** or higher
 - Requires `secret-stack@^6.2.0`
 - Requires `ssb-db2@>=5.0.0`
+- Requires `ssb-box2@>2.0.2`
 - TODO
 
 ```diff
- SecretStack({appKey: require('ssb-caps').shs})
+ const ssb = SecretStack({ caps: require('ssb-caps') })
    .use(require('ssb-master'))
 +  .use(require('ssb-db2'))
    .use(require('ssb-conn'))
++  .use(require('ssb-box2'))
 +  .use(require('ssb-tribes2'))
    .use(require('ssb-blobs'))
    .call(null, config)
+
+ssb.tribes2.start()
+
+ssb.tribes2.create({}, (err, group) => {
+  const content = {
+    type: 'post',
+    text: 'welcome to the group',
+    recps: [group.id]
+  }
+  ssb.db.publishAs(group.subfeed, content, cb)
```

Comment:
We should use `ssb.db.create` not `ssb.db.publishAs`

Reply:
```suggestion
  const keys = group.mySubfeedKeys
  const recps = [group.id]
  const content = { type: 'post', text: 'welcome, friends!' }
  ssb.db.create({ keys, recps, content }, cb)
```

Reply:
What's the difference? I guess I don't fully understand why we need create - apart from being about to explicitly tell it to use box2 encryption?

Reply:
Oh yeah, good point, we also need to add `encryptionFormat: 'box2'` to the opts.

The difference is that publishAs just calls `create` under the hood, and we'd (arj and I) would like to deprecate publish and publishAs in the near future. create() can do everything we need, and it can be misleading to use publish or publishAs because they don't properly support choosing feed formats and encryption formats.

Reply:
@staltz done

Reply:
@Powersource There are two `const content` definitions still in that readme

Reply:
I did this final step